tests/test_ostinato.py

```diff
@@ -186,9 +186,15 @@ def test_extract_seversl_consensus(seed):
         radius, Ts_idx, subseq_idx = stumpy.ostinato(Ts_comp, m)
         consensus_motif = Ts_comp[Ts_idx][subseq_idx : subseq_idx + m].copy()
         for i in range(len(Ts_comp)):
-            nn_idx = np.argmin(stumpy.core.mass(consensus_motif, Ts_comp[i]))
-            Ts_comp[i][nn_idx : nn_idx + m] = np.nan
-            Ts_ref[i][nn_idx : nn_idx + m] = np.nan
+            if i == Ts_idx:
+                # The consensus motif is in the same time series
+                idx = Ts_idx
```

Comment:
@seanlaw 
I am thinking of a situation where we might have several 0 in the distance profile (the output of `mass`). In that case, the value returned by `np.argmin(...)` may not be `Ts_idx`. That's why I created this if-else block.